static/js/components/GcnEventPage.jsx

```diff
@@ -148,25 +253,35 @@ const Localization = ({ loc }) => {
         label={localization.localization_name}
         key={localization.localization_name}
       />
-      <Globe data={localization.contour} />
+      <Globe data={localization.contour?.features} sources={sources} />
     </>
   );
 };
 
 const GcnEventPage = ({ route }) => {
   const mapRef = useRef();
-  const gcnEvent = useSelector((state) => state.gcnEvent);
   const dispatch = useDispatch();
   const styles = useStyles();
 
+  const gcnEvent = useSelector((state) => state.gcnEvent);
+  const gcnEventSources = useSelector((state) => state.sources.gcnEventSources);
+
   useEffect(() => {
     dispatch(gcnEventActions.fetchGcnEvent(route.dateobs));
   }, [route, dispatch]);
 
+  useEffect(() => {
+    dispatch(sourcesActions.fetchGcnEventSources(route.dateobs, {}));
```

Comment:
```suggestion
    dispatch(sourcesActions.fetchGcnEventSources(route.dateobs));
```
(this parameter already defaults to that)